tests/ansible_collections/roles/update-system/tasks/update-system.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Update system
+  yum:
+    name: "*"
+    state: latest
+  register: update
+
+- name: Reboot
+  reboot:
+  when: (ansible_facts['distribution'] == "CentOS") and
```

Comment:
Why only on CentOS Linux?

Reply:
Because in oracle Linux we are doing the reboot after replacing the kernel

Reply:
Ok, please add a comment here so it's clear to others who look at it.